frontend/public/components/monitoring/alerting.tsx

```diff
@@ -324,115 +332,104 @@ export const StateCounts: React.FC<{ alerts: PrometheusAlert[] }> = ({ alerts })
-      </dt>
-      <dd>{t('public~The alert is provided for informational purposes only.')}</dd>
-      <dt>
-        <SeverityIcon severity={AlertSeverity.None} /> <strong>{t('public~None: ')}</strong>
-      </dt>
-      <dd>{t('public~The alert has no defined severity.')}</dd>
-      <dd>
-        {t('public~You can also create custom severity definitions for user workload alerts.')}
-      </dd>
-    </dl>
-  );
-};
+export const severityHelp = (
+  <dl className="co-inline">
+    <dt>
+      <SeverityIcon severity={AlertSeverity.Critical} /> <strong>Critical: </strong>
+    </dt>
+    <dd>
+      The condition that triggered the alert could have a critical impact. The alert requires
+      immediate attention when fired and is typically paged to an individual or to a critical
+      response team.
+    </dd>
+    <dt>
+      <SeverityIcon severity={AlertSeverity.Warning} /> <strong>Warning: </strong>
+    </dt>
+    <dd>
+      The alert provides a warning notification about something that might require attention in
+      order to prevent a problem from occurring. Warnings are typically routed to a ticketing system
+      for non-immediate review.
+    </dd>
+    <dt>
+      <SeverityIcon severity={AlertSeverity.Info} /> <strong>Info: </strong>
+    </dt>
+    <dd>The alert is provided for informational purposes only.</dd>
+    <dt>
+      <SeverityIcon severity={AlertSeverity.None} /> <strong>None: </strong>
+    </dt>
+    <dd>The alert has no defined severity.</dd>
+    <dd>You can also create custom severity definitions for user workload alerts.</dd>
+  </dl>
+);
 
-const SourceHelp: React.FC = () => {
-  const { t } = useTranslation();
-  return (
-    <dl className="co-inline">
-      <dt>
-        <strong>{t('public~Platform: ')}</strong>
-      </dt>
-      <dd>
-        {t(
-          'public~Platform-level alerts relate only to OpenShift namespaces. OpenShift namespaces provide core OpenShift functionality.',
-        )}
-      </dd>
-      <dt>
-        <strong>{t('public~User: ')}</strong>
-      </dt>
-      <dd>
-        {t(
-          'public~User workload alerts relate to user-defined namespaces. These alerts are user-created and are customizable. User workload monitoring can be enabled post-installation to provide observability into your own services.',
-        )}
-      </dd>
-    </dl>
-  );
-};
+export const sourceHelp = (
+  <dl className="co-inline">
+    <dt>
+      <strong>Platform: </strong>
+    </dt>
+    <dd>
+      Platform-level alerts relate only to OpenShift namespaces. OpenShift namespaces provide core
+      OpenShift functionality.
+    </dd>
+    <dt>
+      <strong>User: </strong>
+    </dt>
+    <dd>
+      User workload alerts relate to user-defined namespaces. These alerts are user-created and are
+      customizable. User workload monitoring can be enabled post-installation to provide
+      observability into your own services.
+    </dd>
+  </dl>
+);
```

Comment:
Same as above

Reply:
@divyanshiGupta I did not convert them into functions. I am sorry. I think during rebase with console/master and resolving merge conflict, i did this mistake and accepted an older version. I will rebase and fix this.